### PR TITLE
Fixed Makefile so it would not always attempt to cat pypi creds

### PR DESCRIPTION
**Pull request by johnc1231**
Discussed here: 

https://github.com/hail-is/hail/pull/6318#issuecomment-501313676

### CODE REVIEW

**Comment by tpoterba** [comment]
CI logic is around the assignee, not requested reviewer (still not sure what that means / how the API looks)

File: hail/Makefile [hunk 1]
```diff
@@ -129,9 +129,9 @@ check-pypi:
 HAIL_TWINE_CREDS_FOLDER ?= /secrets/
 
 .PHONY: pypi-deploy
-pypi-deploy: export TWINE_USERNAME := $(shell cat $(HAIL_TWINE_CREDS_FOLDER)/pypi-username)
-pypi-deploy: export TWINE_PASSWORD := $(shell cat $(HAIL_TWINE_CREDS_FOLDER)/pypi-password)
 pypi-deploy: check-pypi wheel
+	export TWINE_USERNAME := $(shell cat $(HAIL_TWINE_CREDS_FOLDER)/pypi-username) \
```
Comment (tpoterba) [inline hail/Makefile]: I think these shouldn't have `export`, this is the same as doing 

```
FOO=1 BAR=2 twine upload
```

Not sure the export is valid bash syntax here.

Comment (johnc1231) [reply hail/Makefile]: Yup, good catch. Fixed.

File: hail/Makefile [hunk 2]
```diff
@@ -129,9 +129,9 @@ check-pypi:
 HAIL_TWINE_CREDS_FOLDER ?= /secrets/
 
 .PHONY: pypi-deploy
-pypi-deploy: export TWINE_USERNAME := $(shell cat $(HAIL_TWINE_CREDS_FOLDER)/pypi-username)
-pypi-deploy: export TWINE_PASSWORD := $(shell cat $(HAIL_TWINE_CREDS_FOLDER)/pypi-password)
 pypi-deploy: check-pypi wheel
+	TWINE_USERNAME := $(shell cat $(HAIL_TWINE_CREDS_FOLDER)/pypi-username) \
+	TWINE_PASSWORD := $(shell cat $(HAIL_TWINE_CREDS_FOLDER)/pypi-password) \
```
Comment (chrisvittal) [inline hail/Makefile]: I don't think this syntax is right. If we're going to use the inside of a rule to run this, then I think that the syntax needs to be shell, which needs to not have spaces or the colon for variable assignment.
```suggestion
	TWINE_PASSWORD=$(shell cat $(HAIL_TWINE_CREDS_FOLDER)/pypi-password) \
```

Comment (tpoterba) [reply hail/Makefile]: oh, cool, neat "suggestion" feature. And yes you're totally right.

Comment (tpoterba) [reply hail/Makefile]: I did have it right the first time: https://github.com/hail-is/hail/pull/6318#issuecomment-501313676

Clearly I am a very thorough reviewer 🙍‍♂

**Pull request merged**